packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+interface AllowedCountries {
+  NorthAmerica: { countryCode: number; format: string };
+  GreatBritain: { countryCode: number; format: string };
+}
+//TODO decide on this number formatting if it is best for the UK
+const countries: AllowedCountries = {
+  NorthAmerica: { countryCode: 1, format: "(###) ###-####" },
+  GreatBritain: { countryCode: 1, format: "(###) #### ######" },
+};
+
+// should it be form field or common form field?
+interface InputPhoneNumberProps extends FormFieldProps {
+  alwaysShowMask?: boolean;
+  country: keyof AllowedCountries;
+  placeholder?: string;
+  showCountryCode?: boolean;
+}
+
+export function InputPhoneNumber({
+  country,
+  placeholder,
+  alwaysShowMask = true,
+  showCountryCode = false,
+  ...rest
+}: InputPhoneNumberProps) {
+  const [value, setValue] = React.useState("");
+  const displayedCountryCode = showCountryCode
+    ? `+${countries[country].countryCode}`
+    : "";
+  const maskingProperties = {
+    allowEmptyFormatting: alwaysShowMask,
+    format: `${displayedCountryCode} ${countries[country].format}`,
+    mask: "_",
```

Comment:
Mockups didn't have a mask, just formatting. Do we actually want the input masked with `_`

Reply:
Great questions, perhaps lets check with Rebecca or Chris on Monday.